TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined

The following error seems to occur on macos.
This did not occur yesterday.

<details>
<summary>full log</summary>

```
Run jurplel/install-qt-action@v3
  with:
    version: 5.15
    host: mac
    target: desktop
    dir: /Users/runner/work/goneovim/goneovim
    modules: qtcharts qtdatavis3d qtpurchasing qtvirtualkeyboard qtwebengine qtnetworkauth qtwebglplugin qtscript
    setup-python: false
    install-deps: true
    cache: false
    cache-key-prefix: install-qt-action
    set-env: true
    tools-only: false
    aqtversion: ==[2](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:2).1.*
    py7zrversion: ==0.19.*
  env:
    cache-version: v6
    GOPATH: /Users/runner/work/goneovim/goneovim
    QT_DIR: /Users/runner/work/goneovim/goneovim/Qt
    QT_VERSION: 5.15
    QT_API: 5.13.0
    QT_DEBUG: false
    GO111MODULE: off
    GOPROXY: https://goproxy.io
    QMAKE_APPLE_DEVICE_ARCHS: arm64
    GOROOT: /Users/runner/hostedtoolcache/go/1.18.7/x64
Run jurplel/install-qt-action/action@v3
  with:
    dir: /Users/runner/work/goneovim/goneovim
    version: 5.15
    host: mac
    target: desktop
    install-deps: true
    modules: qtcharts qtdatavis[3](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:3)d qtpurchasing qtvirtualkeyboard qtwebengine qtnetworkauth qtwebglplugin qtscript
    cache: false
    cache-key-prefix: install-qt-action
    set-env: true
    tools-only: false
    aqtversion: ==2.1.*
    py7zrversion: ==0.19.*
  env:
    cache-version: v6
    GOPATH: /Users/runner/work/goneovim/goneovim
    QT_DIR: /Users/runner/work/goneovim/goneovim/Qt
    QT_VERSION: 5.15
    QT_API: 5.13.0
    QT_DEBUG: false
    GO111MODULE: off
    GOPROXY: https://goproxy.io
    QMAKE_APPLE_DEVICE_ARCHS: arm6[4](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:4)
    GOROOT: /Users/runner/hostedtoolcache/go/1.18.7/x64
/usr/local/bin/brew install p7zip
Warning: p7zip 17.04 is already installed and up-to-date.
To reinstall 17.04, run:
  brew reinstall p7zip
/usr/local/bin/python3 -m pip install setuptools wheel py7zr==0.19.* aqtinstall==2.1.*
Requirement already satisfied: setuptools in /usr/local/lib/python3.10/site-packages (6[5](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:5).3.0)
Requirement already satisfied: wheel in /usr/local/lib/python3.10/site-packages (0.37.1)
Collecting py7zr==0.19.*
  Downloading py7zr-0.19.0-py3-none-any.whl ([6](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:6)5 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.0/65.0 kB [7](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:7).7 MB/s eta 0:00:00
Collecting aqtinstall==2.1.*
  Downloading aqtinstall-2.1.0-py2.py3-none-any.whl (53 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 54.0/54.0 kB 6.9 MB/s eta 0:00:00
Collecting brotli>=1.0.9
  Downloading Brotli-1.0.9-cp310-cp310-macosx_10_9_x[8](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:8)6_64.whl (421 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 421.8/421.8 kB 22.8 MB/s eta 0:00:00
Collecting pybcj>=0.6.0
  Downloading pybcj-1.0.1-cp310-cp310-macosx_10_[9](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:9)_x86_64.whl (23 kB)
Collecting psutil
  Downloading psutil-5.9.2-cp3[10](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:10)-cp310-macosx_10_9_x86_64.whl (239 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 239.3/239.3 kB 19.5 MB/s eta 0:00:00
Collecting pyzstd>=0.14.4
  Downloading pyzstd-0.15.3-cp310-cp310-macosx_10_9_x86_64.whl (391 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 391.3/391.3 kB 24.8 MB/s eta 0:00:00
Collecting texttable
  Downloading texttable-1.6.4-py2.py3-none-any.whl (10 kB)
Collecting multivolumefile>=0.2.3
  Downloading multivolumefile-0.2.3-py3-none-any.whl (17 kB)
Collecting inflate64>=0.1.2
  Downloading inflate64-0.3.0-cp310-cp310-macosx_10_9_x86_64.whl (35 kB)
Collecting pycryptodomex>=3.6.6
  Downloading pycryptodomex-3.15.0-cp35-abi3-macosx_10_9_x86_64.whl (1.6 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 40.9 MB/s eta 0:00:00
Collecting pyppmd<0.19.0,>=0.18.1
  Downloading pyppmd-0.18.3-cp310-cp310-macosx_10_9_x86_64.whl (47 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 47.3/47.3 kB 5.7 MB/s eta 0:00:00
Collecting patch>=1.16
  Downloading patch-1.16.zip (12 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting bs4
  Downloading bs4-0.0.1.tar.gz (1.1 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting requests
  Downloading requests-2.28.1-py3-none-any.whl (62 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.8/62.8 kB 1.1 MB/s eta 0:00:00
Collecting defusedxml
  Downloading defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
Collecting semantic-version
  Downloading semantic_version-2.10.0-py2.py3-none-any.whl (15 kB)
Collecting beautifulsoup4
  Downloading beautifulsoup4-4.[11](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:11).1-py3-none-any.whl ([12](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:12)8 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.2/128.2 kB 17.6 MB/s eta 0:00:00
Collecting urllib3<1.27,>=1.21.1
  Downloading urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.4/140.4 kB 17.0 MB/s eta 0:00:00
Collecting charset-normalizer<3,>=2
  Downloading charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
Collecting certifi>=2017.4.17
  Downloading certifi-2022.9.24-py3-none-any.whl (161 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 161.1/161.1 kB 19.8 MB/s eta 0:00:00
Collecting idna<4,>=2.5
  Downloading idna-3.4-py3-none-any.whl (61 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 9.3 MB/s eta 0:00:00
Collecting soupsieve>1.2
  Downloading soupsieve-2.3.2.post1-py3-none-any.whl (37 kB)
Building wheels for collected packages: patch, bs4
  Building wheel for patch (setup.py): started
  Building wheel for patch (setup.py): finished with status 'done'
  Created wheel for patch: filename=patch-1.16-py3-none-any.whl size=12387 sha256=f5feaa906670b62a98e67384f5329e312ec932535510eb8a3cc9ba32284ab726
  Stored in directory: /Users/runner/Library/Caches/pip/wheels/9e/0b/47/01a46968b6646ca3efa32e23ed436e0c819aca97ec2c[13](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:13)6573
  Building wheel for bs4 (setup.py): started
  Building wheel for bs4 (setup.py): finished with status 'done'
  Created wheel for bs4: filename=bs4-0.0.1-py3-none-any.whl size=1257 sha256=71355cdd3e6cbf11a7d9fe2df[14](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:14)24e6eceac24cec9eb12d1f8053977b4d04b79
  Stored in directory: /Users/runner/Library/Caches/pip/wheels/25/42/45/b773edc52acb16cd2db4cf1a0b47117e2f69bb4eb300ed0e70
Successfully built patch bs4
Installing collected packages: texttable, patch, brotli, urllib3, soupsieve, semantic-version, pyzstd, pyppmd, pycryptodomex, pybcj, psutil, multivolumefile, inflate64, idna, defusedxml, charset-normalizer, certifi, requests, py7zr, beautifulsoup4, bs4, aqtinstall
Successfully installed aqtinstall-2.1.0 beautifulsoup4-4.11.1 brotli-1.0.9 bs4-0.0.1 certifi-2022.9.24 charset-normalizer-2.1.1 defusedxml-0.7.1 idna-3.4 inflate64-0.3.0 multivolumefile-0.2.3 patch-1.16 psutil-5.9.2 py7zr-0.19.0 pybcj-1.0.1 pycryptodomex-3.[15](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:15).0 pyppmd-0.18.3 pyzstd-0.15.3 requests-2.28.1 semantic-version-2.10.0 soupsieve-2.3.2.post1 texttable-1.6.4 urllib3-1.26.12
/usr/local/bin/python3 -m aqt install-qt mac desktop 5.15 --outputdir /Users/runner/work/goneovim/goneovim/Qt --modules qtcharts qtdatavis3d qtpurchasing qtvirtualkeyboard qtwebengine qtnetworkauth qtwebglplugin qtscript
aqtinstall(aqt) v2.1.0 on Python 3.10.7 [CPython Clang 13.0.0 (clang-1300.0.29.30)]
Resolved spec '5.15' to 5.15.2
Downloading qtgraphicaleffects...
Downloading qttranslations...
Downloading qtbase...
Downloading qtmultimedia...
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtgraphicaleffects-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.38643497
Finished installation of qtmultimedia-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.89469308
Downloading qtimageformats...
Finished installation of qttranslations-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.13733[16](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:16)7
Redirected: qt.mirror.constant.com
Downloading qtsensors...
Downloading qtwebchannel...
Finished installation of qtimageformats-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.54495624
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtwebchannel-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.24514623
Downloading qtmacextras...
Finished installation of qtsensors-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.59009562
Redirected: qt.mirror.constant.com
Downloading qtwebsockets...
Downloading qtsvg...
Finished installation of qtmacextras-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.18994164
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Downloading qtxmlpatterns...
Finished installation of qtwebsockets-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.21359297
Finished installation of qtsvg-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.28894567
Redirected: qt.mirror.constant.com
Downloading qttools...
Downloading qt3d...
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtxmlpatterns-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.07094780
Downloading qtlocation...
Redirected: qt.mirror.constant.com
Finished installation of qtlocation-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.68947837
Finished installation of qt3d-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 4.34952822
Downloading qtserialport...
Downloading qtwebview...
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtserialport-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.34340862
Finished installation of qtwebview-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.26050200
Downloading qtscxml...
Downloading qtserialbus...
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtscxml-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.29620646
Finished installation of qtserialbus-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.45879253
Downloading qtgamepad...
Downloading qtremoteobjects...
Finished installation of qttools-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 7.86336907
Redirected: qt.mirror.constant.com
Finished installation of qtgamepad-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.25536222
Redirected: qt.mirror.constant.com
Downloading qtquickcontrols...
Finished installation of qtremoteobjects-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.49008380
Downloading qtspeech...
Redirected: qt.mirror.constant.com
Finished installation of qtbase-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 13.89775746
Redirected: qt.mirror.constant.com
Downloading qtcharts...
Finished installation of qtspeech-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.[17](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:17)561020
Downloading qtconnectivity...
Redirected: qt.mirror.constant.com
Downloading qtnetworkauth...
Finished installation of qtquickcontrols-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.24216829
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtnetworkauth-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.[18](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:18)949239
Finished installation of qtcharts-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.97015853
Downloading qtquickcontrols2...
Finished installation of qtconnectivity-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.60871168
Downloading qtpurchasing...
Downloading qtdatavis3d...
Redirected: qt.mirror.constant.com
Downloading qtdeclarative...
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtpurchasing-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.24056530
Redirected: qt.mirror.constant.com
Finished installation of qtdatavis3d-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.44273222
Downloading qtscript...
Finished installation of qtquickcontrols2-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.35696787
Downloading qtvirtualkeyboard...
Redirected: qt.mirror.constant.com
Redirected: qt.mirror.constant.com
Finished installation of qtscript-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.62698364
Finished installation of qtvirtualkeyboard-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.78244021
Downloading qtwebengine...
Redirected: qt.mirror.constant.com
Finished installation of qtdeclarative-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 4.82415925
Finished installation of qtwebengine-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 12.414[20](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:20)156
Downloading qtwebglplugin...
Redirected: qt.mirror.constant.com
Finished installation of qtwebglplugin-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.29725365
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/bin/qmake
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DRender.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DAnimation.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Xml.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Network.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuick.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Multimedia.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5OpenGLExtensions.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickTest.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Script.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Widgets.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DLogic.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Purchasing.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5RemoteObjects.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DCore.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickInput.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5MultimediaWidgets.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebChannel.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5ScriptTools.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5NetworkAuth.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DExtras.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5DBus.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5DataVisualization.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Nfc.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QmlWorkerScript.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5UiPlugin.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickAnimation.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5MacExtras.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5OpenGL.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5XmlPatterns.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Core.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebView.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Gamepad.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Test.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QmlModels.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Gui.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5VirtualKeyboard.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5SerialPort.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5UiTools.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickExtras.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Sensors.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5TextToSpeech.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5RepParser.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Sql.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickRender.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickScene2D.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebSockets.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Qml.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5SerialBus.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Positioning.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Charts.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Location.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebEngineCore.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Bluetooth.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebEngineWidgets.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Quick.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebEngine.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5PrintSupport.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Designer.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickControls2.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Scxml.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Help.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickTemplates2.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5PositioningQuick.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Svg.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DInput.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickWidgets.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Concurrent.pc
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5ClipboardSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5QmlDebug.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5GraphicsSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5PlatformCompositorSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5ServiceSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5EdidSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5DeviceDiscoverySupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5EventDispatcherSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5QmlDevTools.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5FontDatabaseSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5OpenGLExtensions.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5ThemeSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5UiTools.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5FbSupport.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5Bootstrap.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5PacketProtocol.la
Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5AccessibilitySupport.la
Finished installation
Time elapsed: 36.[21](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:21)5433[32](https://github.com/akiyosi/goneovim/actions/runs/3214592020/jobs/5255087339#step:3:33) second
Error: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
```
</details>

Well, ```Error: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined``` doesn't look like a Python exception, but from a JavaScript runtime, so posting here is not wrong, I think.

<br>

\* Since the log posted by the OP contains some irrelevant content, I saved the full log of [test-and-build-macos-11 (1.18.x, macos-11, 5.15) | fixup · akiyosi/goneovim@fdc6b1a](https://github.com/akiyosi/goneovim/actions/runs/3247244936/jobs/5326975523) (it's the latest failed run, though) and post it here:

<details>
<summary>full log</summary>

```log
2022-10-14T03:31:39.0646192Z Requested labels: macos-11
2022-10-14T03:31:39.0646253Z Job defined at: akiyosi/goneovim/.github/workflows/ci.yaml@refs/heads/improve-linux-ci
2022-10-14T03:31:39.0646278Z Waiting for a runner to pick up this job...
2022-10-14T03:31:39.4109118Z Job is waiting for a hosted runner to come online.
2022-10-14T03:31:46.8729007Z Job is about to start running on the hosted runner: GitHub Actions 4 (hosted)
2022-10-14T03:31:50.1240430Z Current runner version: '2.298.2'
2022-10-14T03:31:50.1319220Z ##[group]Operating System
2022-10-14T03:31:50.1319890Z macOS
2022-10-14T03:31:50.1320210Z 11.7
2022-10-14T03:31:50.1320600Z 20G817
2022-10-14T03:31:50.1320880Z ##[endgroup]
2022-10-14T03:31:50.1321220Z ##[group]Runner Image
2022-10-14T03:31:50.1321690Z Image: macos-11
2022-10-14T03:31:50.1321980Z Version: 20221010.1
2022-10-14T03:31:50.1322500Z Included Software: https://github.com/actions/runner-images/blob/macOS-11/20221010.1/images/macos/macos-11-Readme.md
2022-10-14T03:31:50.1323160Z Image Release: https://github.com/actions/runner-images/releases/tag/macOS-11%2F20221010.1
2022-10-14T03:31:50.1323580Z ##[endgroup]
2022-10-14T03:31:50.1324020Z ##[group]Runner Image Provisioner
2022-10-14T03:31:50.1324380Z 2.0.78.1
2022-10-14T03:31:50.1324640Z ##[endgroup]
2022-10-14T03:31:50.1325660Z ##[group]GITHUB_TOKEN Permissions
2022-10-14T03:31:50.1326410Z Actions: write
2022-10-14T03:31:50.1326890Z Checks: write
2022-10-14T03:31:50.1327350Z Contents: write
2022-10-14T03:31:50.1327890Z Deployments: write
2022-10-14T03:31:50.1328260Z Discussions: write
2022-10-14T03:31:50.1328760Z Issues: write
2022-10-14T03:31:50.1329100Z Metadata: read
2022-10-14T03:31:50.1329430Z Packages: write
2022-10-14T03:31:50.1329810Z Pages: write
2022-10-14T03:31:50.1330160Z PullRequests: write
2022-10-14T03:31:50.1330600Z RepositoryProjects: write
2022-10-14T03:31:50.1330930Z SecurityEvents: write
2022-10-14T03:31:50.1331270Z Statuses: write
2022-10-14T03:31:50.1331700Z ##[endgroup]
2022-10-14T03:31:50.1336250Z Secret source: Actions
2022-10-14T03:31:50.1336950Z Prepare workflow directory
2022-10-14T03:31:50.2955960Z Prepare all required actions
2022-10-14T03:31:50.3201590Z Getting action download info
2022-10-14T03:31:50.5850890Z Download action repository 'actions/setup-go@v1' (SHA:0caeaed6fd66a828038c2da3c0f662a42862658f)
2022-10-14T03:31:52.8023500Z Download action repository 'jurplel/install-qt-action@v3' (SHA:43ec12788e42f375acfcb2cec059edfb9572fbaa)
2022-10-14T03:31:56.5779950Z Download action repository 'actions/checkout@v2' (SHA:e2f20e631ae6d7dd3b768f56a5d2af784dd54791)
2022-10-14T03:31:57.9289160Z Download action repository 'actions/cache@v2' (SHA:937d24475381cd9c75ae6db12cb4e79714b926ed)
2022-10-14T03:31:59.3373330Z Download action repository 'actions/upload-artifact@v1' (SHA:3446296876d12d4e3a0f3145a3c87e67bf0a16b5)
2022-10-14T03:32:00.4603420Z Getting action download info
2022-10-14T03:32:00.9224140Z Download action repository 'actions/setup-python@v4' (SHA:13ae5bb136fac2878aff31522b9efb785519f984)
2022-10-14T03:32:02.5793130Z ##[group]Run actions/setup-go@v1
2022-10-14T03:32:02.5793500Z with:
2022-10-14T03:32:02.5793700Z   go-version: 1.18.x
2022-10-14T03:32:02.5793920Z env:
2022-10-14T03:32:02.5794110Z   cache-version: v7
2022-10-14T03:32:02.5794360Z   GOPATH: /Users/runner/work/goneovim/goneovim
2022-10-14T03:32:02.5794620Z   QT_DIR: /Users/runner/work/goneovim/goneovim/Qt
2022-10-14T03:32:02.5794850Z   QT_VERSION: 5.15
2022-10-14T03:32:02.5795050Z   QT_API: 5.13.0
2022-10-14T03:32:02.5795320Z   QT_DEBUG: false
2022-10-14T03:32:02.5795510Z   GO111MODULE: off
2022-10-14T03:32:02.5795750Z   GOPROXY: https://goproxy.io
2022-10-14T03:32:02.5795990Z   QMAKE_APPLE_DEVICE_ARCHS: arm64
2022-10-14T03:32:02.5796200Z ##[endgroup]
2022-10-14T03:32:03.9980520Z ##[group]Run jurplel/install-qt-action@v3
2022-10-14T03:32:03.9980940Z with:
2022-10-14T03:32:03.9981180Z   version: 5.15
2022-10-14T03:32:03.9981410Z   host: mac
2022-10-14T03:32:03.9981800Z   target: desktop
2022-10-14T03:32:03.9982110Z   dir: /Users/runner/work/goneovim/goneovim
2022-10-14T03:32:03.9982500Z   modules: qtcharts qtdatavis3d qtpurchasing qtvirtualkeyboard qtwebengine qtnetworkauth qtwebglplugin qtscript
2022-10-14T03:32:03.9982900Z   setup-python: false
2022-10-14T03:32:03.9983160Z   install-deps: true
2022-10-14T03:32:03.9983550Z   cache: false
2022-10-14T03:32:03.9983880Z   cache-key-prefix: install-qt-action
2022-10-14T03:32:03.9984170Z   set-env: true
2022-10-14T03:32:03.9984380Z   tools-only: false
2022-10-14T03:32:03.9985310Z   aqtversion: ==2.1.*
2022-10-14T03:32:03.9985710Z   py7zrversion: ==0.19.*
2022-10-14T03:32:03.9985930Z env:
2022-10-14T03:32:03.9986170Z   cache-version: v7
2022-10-14T03:32:03.9986460Z   GOPATH: /Users/runner/work/goneovim/goneovim
2022-10-14T03:32:03.9986930Z   QT_DIR: /Users/runner/work/goneovim/goneovim/Qt
2022-10-14T03:32:03.9987260Z   QT_VERSION: 5.15
2022-10-14T03:32:03.9987500Z   QT_API: 5.13.0
2022-10-14T03:32:03.9987720Z   QT_DEBUG: false
2022-10-14T03:32:03.9987960Z   GO111MODULE: off
2022-10-14T03:32:03.9988370Z   GOPROXY: https://goproxy.io
2022-10-14T03:32:03.9988640Z   QMAKE_APPLE_DEVICE_ARCHS: arm64
2022-10-14T03:32:03.9988960Z   GOROOT: /Users/runner/hostedtoolcache/go/1.18.7/x64
2022-10-14T03:32:03.9989260Z ##[endgroup]
2022-10-14T03:32:04.0363360Z ##[group]Run jurplel/install-qt-action/action@v3
2022-10-14T03:32:04.0363730Z with:
2022-10-14T03:32:04.0364030Z   dir: /Users/runner/work/goneovim/goneovim
2022-10-14T03:32:04.0364290Z   version: 5.15
2022-10-14T03:32:04.0364580Z   host: mac
2022-10-14T03:32:04.0365030Z   target: desktop
2022-10-14T03:32:04.0365530Z   install-deps: true
2022-10-14T03:32:04.0365910Z   modules: qtcharts qtdatavis3d qtpurchasing qtvirtualkeyboard qtwebengine qtnetworkauth qtwebglplugin qtscript
2022-10-14T03:32:04.0399760Z   cache: false
2022-10-14T03:32:04.0400120Z   cache-key-prefix: install-qt-action
2022-10-14T03:32:04.0400380Z   set-env: true
2022-10-14T03:32:04.0400630Z   tools-only: false
2022-10-14T03:32:04.0400880Z   aqtversion: ==2.1.*
2022-10-14T03:32:04.0401080Z   py7zrversion: ==0.19.*
2022-10-14T03:32:04.0401300Z env:
2022-10-14T03:32:04.0401530Z   cache-version: v7
2022-10-14T03:32:04.0401800Z   GOPATH: /Users/runner/work/goneovim/goneovim
2022-10-14T03:32:04.0402110Z   QT_DIR: /Users/runner/work/goneovim/goneovim/Qt
2022-10-14T03:32:04.0402360Z   QT_VERSION: 5.15
2022-10-14T03:32:04.0402590Z   QT_API: 5.13.0
2022-10-14T03:32:04.0402810Z   QT_DEBUG: false
2022-10-14T03:32:04.0403030Z   GO111MODULE: off
2022-10-14T03:32:04.0403310Z   GOPROXY: https://goproxy.io
2022-10-14T03:32:04.0403580Z   QMAKE_APPLE_DEVICE_ARCHS: arm64
2022-10-14T03:32:04.0403860Z   GOROOT: /Users/runner/hostedtoolcache/go/1.18.7/x64
2022-10-14T03:32:04.0404130Z ##[endgroup]
2022-10-14T03:32:05.6690840Z [command]/usr/local/bin/brew install p7zip
2022-10-14T03:32:11.6479120Z [33mWarning:[0m p7zip 17.04 is already installed and up-to-date.
2022-10-14T03:32:11.6480170Z To reinstall 17.04, run:
2022-10-14T03:32:11.6481590Z   brew reinstall p7zip
2022-10-14T03:32:11.6595850Z [command]/usr/local/bin/python3 -m pip install setuptools wheel py7zr==0.19.* aqtinstall==2.1.*
2022-10-14T03:32:14.3646780Z Requirement already satisfied: setuptools in /usr/local/lib/python3.10/site-packages (65.3.0)
2022-10-14T03:32:14.3652950Z Requirement already satisfied: wheel in /usr/local/lib/python3.10/site-packages (0.37.1)
2022-10-14T03:32:14.6265610Z Collecting py7zr==0.19.*
2022-10-14T03:32:14.6737080Z   Downloading py7zr-0.19.0-py3-none-any.whl (65 kB)
2022-10-14T03:32:14.6908500Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 65.0/65.0 kB 4.2 MB/s eta 0:00:00
2022-10-14T03:32:14.8538570Z Collecting aqtinstall==2.1.*
2022-10-14T03:32:14.8652980Z   Downloading aqtinstall-2.1.0-py2.py3-none-any.whl (53 kB)
2022-10-14T03:32:14.8758680Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 54.0/54.0 kB 6.1 MB/s eta 0:00:00
2022-10-14T03:32:15.0120370Z Collecting brotli>=1.0.9
2022-10-14T03:32:15.0252250Z   Downloading Brotli-1.0.9-cp310-cp310-macosx_10_9_x86_64.whl (421 kB)
2022-10-14T03:32:15.0459670Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 421.8/421.8 kB 24.0 MB/s eta 0:00:00
2022-10-14T03:32:15.1607510Z Collecting multivolumefile>=0.2.3
2022-10-14T03:32:15.1726590Z   Downloading multivolumefile-0.2.3-py3-none-any.whl (17 kB)
2022-10-14T03:32:15.3575980Z Collecting inflate64>=0.1.2
2022-10-14T03:32:15.3713280Z   Downloading inflate64-0.3.0-cp310-cp310-macosx_10_9_x86_64.whl (35 kB)
2022-10-14T03:32:15.6279120Z Collecting psutil
2022-10-14T03:32:15.6452400Z   Downloading psutil-5.9.2-cp310-cp310-macosx_10_9_x86_64.whl (239 kB)
2022-10-14T03:32:15.6607370Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 239.3/239.3 kB 19.0 MB/s eta 0:00:00
2022-10-14T03:32:15.9942890Z Collecting pycryptodomex>=3.6.6
2022-10-14T03:32:16.0110300Z   Downloading pycryptodomex-3.15.0-cp35-abi3-macosx_10_9_x86_64.whl (1.6 MB)
2022-10-14T03:32:16.1091550Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.6/1.6 MB 16.3 MB/s eta 0:00:00
2022-10-14T03:32:16.4931120Z Collecting pyppmd<0.19.0,>=0.18.1
2022-10-14T03:32:16.5049830Z   Downloading pyppmd-0.18.3-cp310-cp310-macosx_10_9_x86_64.whl (47 kB)
2022-10-14T03:32:16.5134960Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 47.3/47.3 kB 6.4 MB/s eta 0:00:00
2022-10-14T03:32:16.5394940Z Collecting texttable
2022-10-14T03:32:16.5501030Z   Downloading texttable-1.6.4-py2.py3-none-any.whl (10 kB)
2022-10-14T03:32:16.7802650Z Collecting pybcj>=0.6.0
2022-10-14T03:32:16.8352760Z   Downloading pybcj-1.0.1-cp310-cp310-macosx_10_9_x86_64.whl (23 kB)
2022-10-14T03:32:17.0494200Z Collecting pyzstd>=0.14.4
2022-10-14T03:32:17.0735980Z   Downloading pyzstd-0.15.3-cp310-cp310-macosx_10_9_x86_64.whl (391 kB)
2022-10-14T03:32:17.0882300Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 391.3/391.3 kB 33.5 MB/s eta 0:00:00
2022-10-14T03:32:17.2002760Z Collecting requests
2022-10-14T03:32:17.2107730Z   Downloading requests-2.28.1-py3-none-any.whl (62 kB)
2022-10-14T03:32:17.2205550Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 62.8/62.8 kB 8.3 MB/s eta 0:00:00
2022-10-14T03:32:17.2470810Z Collecting bs4
2022-10-14T03:32:17.2570160Z   Downloading bs4-0.0.1.tar.gz (1.1 kB)
2022-10-14T03:32:17.2728970Z   Preparing metadata (setup.py): started
2022-10-14T03:32:17.7091240Z   Preparing metadata (setup.py): finished with status 'done'
2022-10-14T03:32:17.7474640Z Collecting defusedxml
2022-10-14T03:32:17.7588120Z   Downloading defusedxml-0.7.1-py2.py3-none-any.whl (25 kB)
2022-10-14T03:32:17.8031170Z Collecting semantic-version
2022-10-14T03:32:17.8134900Z   Downloading semantic_version-2.10.0-py2.py3-none-any.whl (15 kB)
2022-10-14T03:32:17.8419400Z Collecting patch>=1.16
2022-10-14T03:32:17.8554830Z   Downloading patch-1.16.zip (12 kB)
2022-10-14T03:32:17.8644520Z   Preparing metadata (setup.py): started
2022-10-14T03:32:18.1948520Z   Preparing metadata (setup.py): finished with status 'done'
2022-10-14T03:32:18.3689700Z Collecting beautifulsoup4
2022-10-14T03:32:18.3795300Z   Downloading beautifulsoup4-4.11.1-py3-none-any.whl (128 kB)
2022-10-14T03:32:18.3899400Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 128.2/128.2 kB 17.4 MB/s eta 0:00:00
2022-10-14T03:32:18.4733580Z Collecting certifi>=2017.4.17
2022-10-14T03:32:18.4836760Z   Downloading certifi-2022.9.24-py3-none-any.whl (161 kB)
2022-10-14T03:32:18.5064900Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 161.1/161.1 kB 7.7 MB/s eta 0:00:00
2022-10-14T03:32:18.5830090Z Collecting urllib3<1.27,>=1.21.1
2022-10-14T03:32:18.5939670Z   Downloading urllib3-1.26.12-py2.py3-none-any.whl (140 kB)
2022-10-14T03:32:18.6041630Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 140.4/140.4 kB 18.7 MB/s eta 0:00:00
2022-10-14T03:32:18.6903610Z Collecting charset-normalizer<3,>=2
2022-10-14T03:32:18.7005420Z   Downloading charset_normalizer-2.1.1-py3-none-any.whl (39 kB)
2022-10-14T03:32:18.7476390Z Collecting idna<4,>=2.5
2022-10-14T03:32:18.7576020Z   Downloading idna-3.4-py3-none-any.whl (61 kB)
2022-10-14T03:32:18.7665570Z      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 61.5/61.5 kB 8.7 MB/s eta 0:00:00
2022-10-14T03:32:18.8916540Z Collecting soupsieve>1.2
2022-10-14T03:32:18.9016060Z   Downloading soupsieve-2.3.2.post1-py3-none-any.whl (37 kB)
2022-10-14T03:32:18.9761290Z Building wheels for collected packages: patch, bs4
2022-10-14T03:32:18.9771110Z   Building wheel for patch (setup.py): started
2022-10-14T03:32:19.4523250Z   Building wheel for patch (setup.py): finished with status 'done'
2022-10-14T03:32:19.4535260Z   Created wheel for patch: filename=patch-1.16-py3-none-any.whl size=12387 sha256=8d09f802474be9247c173f16fb0234715a7497d685c6a9b72e68fee693f9fdbd
2022-10-14T03:32:19.4538920Z   Stored in directory: /Users/runner/Library/Caches/pip/wheels/9e/0b/47/01a46968b6646ca3efa32e23ed436e0c819aca97ec2c136573
2022-10-14T03:32:19.4585350Z   Building wheel for bs4 (setup.py): started
2022-10-14T03:32:19.8832690Z   Building wheel for bs4 (setup.py): finished with status 'done'
2022-10-14T03:32:19.8842590Z   Created wheel for bs4: filename=bs4-0.0.1-py3-none-any.whl size=1257 sha256=a79edd08d77a3598bd946cdcbc032325208f4f94b13ba02e60c30fdb39479b45
2022-10-14T03:32:19.8845460Z   Stored in directory: /Users/runner/Library/Caches/pip/wheels/25/42/45/b773edc52acb16cd2db4cf1a0b47117e2f69bb4eb300ed0e70
2022-10-14T03:32:19.8881150Z Successfully built patch bs4
2022-10-14T03:32:20.0993660Z Installing collected packages: texttable, patch, brotli, urllib3, soupsieve, semantic-version, pyzstd, pyppmd, pycryptodomex, pybcj, psutil, multivolumefile, inflate64, idna, defusedxml, charset-normalizer, certifi, requests, py7zr, beautifulsoup4, bs4, aqtinstall
2022-10-14T03:32:21.6093150Z Successfully installed aqtinstall-2.1.0 beautifulsoup4-4.11.1 brotli-1.0.9 bs4-0.0.1 certifi-2022.9.24 charset-normalizer-2.1.1 defusedxml-0.7.1 idna-3.4 inflate64-0.3.0 multivolumefile-0.2.3 patch-1.16 psutil-5.9.2 py7zr-0.19.0 pybcj-1.0.1 pycryptodomex-3.15.0 pyppmd-0.18.3 pyzstd-0.15.3 requests-2.28.1 semantic-version-2.10.0 soupsieve-2.3.2.post1 texttable-1.6.4 urllib3-1.26.12
2022-10-14T03:32:22.0228000Z [command]/usr/local/bin/python3 -m aqt install-qt mac desktop 5.15 --outputdir /Users/runner/work/goneovim/goneovim/Qt --modules qtcharts qtdatavis3d qtpurchasing qtvirtualkeyboard qtwebengine qtnetworkauth qtwebglplugin qtscript
2022-10-14T03:32:24.7894150Z aqtinstall(aqt) v2.1.0 on Python 3.10.7 [CPython Clang 13.0.0 (clang-1300.0.29.30)]
2022-10-14T03:32:25.6935370Z Resolved spec '5.15' to 5.15.2
2022-10-14T03:32:29.4174880Z Downloading qtgraphicaleffects...
2022-10-14T03:32:29.4277390Z Downloading qtbase...
2022-10-14T03:32:29.4485460Z Downloading qtmultimedia...
2022-10-14T03:32:29.4688890Z Downloading qttranslations...
2022-10-14T03:32:30.2107320Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:32:30.2182500Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:32:30.2183410Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:32:30.2184330Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:32:36.7563180Z Finished installation of qtgraphicaleffects-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 8.13697118
2022-10-14T03:32:37.7162280Z Downloading qtimageformats...
2022-10-14T03:32:38.4760440Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:32:42.7268160Z Finished installation of qtimageformats-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 5.95850726
2022-10-14T03:32:43.6803950Z Downloading qtmacextras...
2022-10-14T03:32:44.4121520Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:32:47.0427290Z Finished installation of qtmacextras-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 4.30716682
2022-10-14T03:32:47.8139680Z Downloading qtxmlpatterns...
2022-10-14T03:32:48.5459470Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:33:05.5182440Z Finished installation of qttranslations-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 36.85384361
2022-10-14T03:33:05.5307860Z Finished installation of qtmultimedia-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 36.89830240
2022-10-14T03:33:06.4439700Z Downloading qtwebchannel...
2022-10-14T03:33:06.4888030Z Downloading qtsensors...
2022-10-14T03:33:07.1721580Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:33:07.2180550Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:33:12.7173190Z Finished installation of qtwebchannel-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 7.19484267
2022-10-14T03:33:13.6759080Z Downloading qtwebsockets...
2022-10-14T03:33:14.4084980Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:33:23.5300850Z Finished installation of qtwebsockets-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 10.80573637
2022-10-14T03:33:24.4872650Z Downloading qttools...
2022-10-14T03:33:25.2511180Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:33:33.3703280Z Finished installation of qtsensors-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 27.83286594
2022-10-14T03:33:34.3206620Z Downloading qtsvg...
2022-10-14T03:33:35.0498940Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:33:38.0479980Z Finished installation of qtsvg-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 4.64554898
2022-10-14T03:33:38.1190590Z Finished installation of qtxmlpatterns-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 51.04718181
2022-10-14T03:33:38.8399190Z Downloading qtlocation...
2022-10-14T03:33:38.9008430Z Downloading qt3d...
2022-10-14T03:33:39.5731620Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:33:39.6349390Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:34:26.0728480Z Finished installation of qttools-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 62.53587034
2022-10-14T03:34:27.0358060Z Downloading qtquickcontrols...
2022-10-14T03:34:27.8034990Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:34:28.9947320Z Finished installation of qtlocation-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 50.88721934
2022-10-14T03:34:30.0084610Z Downloading qtserialport...
2022-10-14T03:34:30.7421170Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:34:34.3080250Z Finished installation of qtserialport-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 5.30625568
2022-10-14T03:34:35.3067170Z Downloading qtscxml...
2022-10-14T03:34:36.0655780Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:34:50.5764980Z Finished installation of qtscxml-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 16.25655958
2022-10-14T03:34:52.0062920Z Downloading qtgamepad...
2022-10-14T03:34:53.0403620Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:34:58.2807350Z Finished installation of qtgamepad-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 7.69705173
2022-10-14T03:34:59.3545770Z Downloading qtspeech...
2022-10-14T03:35:00.0883490Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:00.5275270Z Finished installation of qtspeech-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.23984398
2022-10-14T03:35:00.6912790Z Finished installation of qtquickcontrols-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 34.61122070
2022-10-14T03:35:01.2912260Z Downloading qtremoteobjects...
2022-10-14T03:35:01.4970860Z Downloading qtquickcontrols2...
2022-10-14T03:35:02.0232090Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:02.2247730Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:07.6992520Z Finished installation of qtremoteobjects-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 7.13146440
2022-10-14T03:35:07.9524950Z Finished installation of qt3d-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 89.90278893
2022-10-14T03:35:08.4603950Z Downloading qtcharts...
2022-10-14T03:35:08.6938270Z Downloading qtwebview...
2022-10-14T03:35:08.9670540Z Finished installation of qtquickcontrols2-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 8.25609653
2022-10-14T03:35:09.1856130Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:09.4736280Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:09.8259550Z Downloading qtnetworkauth...
2022-10-14T03:35:10.0763050Z Finished installation of qtwebview-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.11423219
2022-10-14T03:35:10.5552170Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:10.5612810Z Finished installation of qtcharts-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 2.84706515
2022-10-14T03:35:10.7494330Z Finished installation of qtnetworkauth-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.77289906
2022-10-14T03:35:10.8788940Z Downloading qtserialbus...
2022-10-14T03:35:11.3222290Z Downloading qtdatavis3d...
2022-10-14T03:35:11.5032470Z Downloading qtpurchasing...
2022-10-14T03:35:11.6824980Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:12.0998750Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:12.2323430Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:18.9621210Z Finished installation of qtpurchasing-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 8.19976137
2022-10-14T03:35:19.4450870Z Finished installation of qtserialbus-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 9.36153961
2022-10-14T03:35:19.9440880Z Downloading qtscript...
2022-10-14T03:35:20.3981340Z Downloading qtvirtualkeyboard...
2022-10-14T03:35:20.6512980Z Finished installation of qtdatavis3d-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 10.08047466
2022-10-14T03:35:20.6806880Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:21.1304170Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:24.6283800Z Finished installation of qtscript-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 5.64856112
2022-10-14T03:35:24.7679920Z Finished installation of qtvirtualkeyboard-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 5.31130940
2022-10-14T03:35:25.0847340Z Finished installation of qtbase-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 176.49237736
2022-10-14T03:35:25.5563920Z Downloading qtwebengine...
2022-10-14T03:35:25.8933500Z Downloading qtconnectivity...
2022-10-14T03:35:26.2911540Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:26.6308020Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:48.6321950Z Finished installation of qtconnectivity-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 23.53977207
2022-10-14T03:35:49.4854540Z Downloading qtdeclarative...
2022-10-14T03:35:50.2110060Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:57.2120680Z Finished installation of qtwebengine-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 32.42827243
2022-10-14T03:35:57.2151020Z Finished installation of qtdeclarative-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 8.56394499
2022-10-14T03:35:57.9475290Z Downloading qtwebglplugin...
2022-10-14T03:35:58.7114080Z Redirected: mirrors.ocf.berkeley.edu
2022-10-14T03:35:59.1690010Z Finished installation of qtwebglplugin-MacOS-MacOS_10_13-Clang-MacOS-MacOS_10_13-X86_64.7z in 1.95298752
2022-10-14T03:35:59.6315530Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/bin/qmake
2022-10-14T03:35:59.6438500Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DRender.pc
2022-10-14T03:35:59.7847920Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DAnimation.pc
2022-10-14T03:35:59.9092870Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Xml.pc
2022-10-14T03:36:00.0352910Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Network.pc
2022-10-14T03:36:00.1596900Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuick.pc
2022-10-14T03:36:00.3662920Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Multimedia.pc
2022-10-14T03:36:00.4913720Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5OpenGLExtensions.pc
2022-10-14T03:36:00.6152400Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickTest.pc
2022-10-14T03:36:00.7407430Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Script.pc
2022-10-14T03:36:00.8641330Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Widgets.pc
2022-10-14T03:36:00.9883360Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DLogic.pc
2022-10-14T03:36:01.1131440Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Purchasing.pc
2022-10-14T03:36:01.2361430Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5RemoteObjects.pc
2022-10-14T03:36:01.3729420Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DCore.pc
2022-10-14T03:36:01.4975510Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickInput.pc
2022-10-14T03:36:01.6204650Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5MultimediaWidgets.pc
2022-10-14T03:36:01.7433670Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebChannel.pc
2022-10-14T03:36:01.8664200Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5ScriptTools.pc
2022-10-14T03:36:01.9895600Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5NetworkAuth.pc
2022-10-14T03:36:02.1125430Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DExtras.pc
2022-10-14T03:36:02.2350220Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5DBus.pc
2022-10-14T03:36:02.3575700Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5DataVisualization.pc
2022-10-14T03:36:02.4810850Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Nfc.pc
2022-10-14T03:36:02.6050860Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QmlWorkerScript.pc
2022-10-14T03:36:02.7283570Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5UiPlugin.pc
2022-10-14T03:36:02.8517960Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickAnimation.pc
2022-10-14T03:36:02.9735990Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5MacExtras.pc
2022-10-14T03:36:03.0969360Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5OpenGL.pc
2022-10-14T03:36:03.2197240Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5XmlPatterns.pc
2022-10-14T03:36:03.3416340Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Core.pc
2022-10-14T03:36:03.4646210Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebView.pc
2022-10-14T03:36:03.5870920Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Gamepad.pc
2022-10-14T03:36:03.7094370Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Test.pc
2022-10-14T03:36:03.8327640Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QmlModels.pc
2022-10-14T03:36:03.9558160Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Gui.pc
2022-10-14T03:36:04.0811810Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5VirtualKeyboard.pc
2022-10-14T03:36:04.2044720Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5SerialPort.pc
2022-10-14T03:36:04.3302100Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5UiTools.pc
2022-10-14T03:36:04.4540430Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickExtras.pc
2022-10-14T03:36:04.5790230Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Sensors.pc
2022-10-14T03:36:04.7025550Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5TextToSpeech.pc
2022-10-14T03:36:04.8278630Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5RepParser.pc
2022-10-14T03:36:04.9519380Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Sql.pc
2022-10-14T03:36:05.0772870Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickRender.pc
2022-10-14T03:36:05.2025100Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DQuickScene2D.pc
2022-10-14T03:36:05.3261600Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebSockets.pc
2022-10-14T03:36:05.4484530Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Qml.pc
2022-10-14T03:36:05.5720570Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5SerialBus.pc
2022-10-14T03:36:05.6933670Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Positioning.pc
2022-10-14T03:36:05.8158980Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Charts.pc
2022-10-14T03:36:05.9388350Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Location.pc
2022-10-14T03:36:06.0646190Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebEngineCore.pc
2022-10-14T03:36:06.1898350Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Bluetooth.pc
2022-10-14T03:36:06.3129930Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebEngineWidgets.pc
2022-10-14T03:36:06.4366750Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Quick.pc
2022-10-14T03:36:06.5598810Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5WebEngine.pc
2022-10-14T03:36:06.6832110Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5PrintSupport.pc
2022-10-14T03:36:06.8075690Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Designer.pc
2022-10-14T03:36:06.9308700Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickControls2.pc
2022-10-14T03:36:07.0541520Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Scxml.pc
2022-10-14T03:36:07.1800590Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Help.pc
2022-10-14T03:36:07.3043040Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickTemplates2.pc
2022-10-14T03:36:07.4290790Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5PositioningQuick.pc
2022-10-14T03:36:07.5547850Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Svg.pc
2022-10-14T03:36:07.6795060Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt53DInput.pc
2022-10-14T03:36:07.8041640Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5QuickWidgets.pc
2022-10-14T03:36:07.9286600Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/pkgconfig/Qt5Concurrent.pc
2022-10-14T03:36:08.0537360Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5ClipboardSupport.la
2022-10-14T03:36:08.4253350Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5QmlDebug.la
2022-10-14T03:36:08.6357290Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5GraphicsSupport.la
2022-10-14T03:36:08.6385080Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5PlatformCompositorSupport.la
2022-10-14T03:36:08.6411250Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5ServiceSupport.la
2022-10-14T03:36:08.6441130Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5EdidSupport.la
2022-10-14T03:36:08.6473540Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5DeviceDiscoverySupport.la
2022-10-14T03:36:08.6507800Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5EventDispatcherSupport.la
2022-10-14T03:36:08.6534620Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5QmlDevTools.la
2022-10-14T03:36:08.6568560Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5FontDatabaseSupport.la
2022-10-14T03:36:08.6595130Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5OpenGLExtensions.la
2022-10-14T03:36:08.6621300Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5ThemeSupport.la
2022-10-14T03:36:08.6646900Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5UiTools.la
2022-10-14T03:36:08.7048300Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5FbSupport.la
2022-10-14T03:36:08.7080260Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5Bootstrap.la
2022-10-14T03:36:08.7501940Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5PacketProtocol.la
2022-10-14T03:36:08.7930050Z Patching /Users/runner/work/goneovim/goneovim/Qt/5.15.2/clang_64/lib/libQt5AccessibilitySupport.la
2022-10-14T03:36:08.8358820Z Finished installation
2022-10-14T03:36:08.8359610Z Time elapsed: 224.05159844 second
2022-10-14T03:36:08.9419590Z ##[error]TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
2022-10-14T03:36:09.0873250Z Post job cleanup.
2022-10-14T03:36:09.1893060Z Cleaning up orphan processes

```

</details>

Maybe the config [YAML file](https://github.com/akiyosi/goneovim/blob/fdc6b1a7ec5e504ea1f7549c926efa7800455e94/.github/workflows/ci.yaml) is also helpful:

<details>
<summary>YAML (effective part)</summary>

```YAML
  test-and-build-macos-11:
    strategy:
      matrix:
        go-version: [1.18.x]
        platform: [macos-11]
        qtversion: [5.15]
    runs-on: ${{ matrix.platform }}
    env:
      GOPATH: ${{ github.workspace }}
      QT_DIR: ${{ github.workspace }}/Qt
      QT_VERSION: ${{ matrix.qtversion }}
      QT_API: 5.13.0
      QT_DEBUG: false
      GO111MODULE: off
      GOPROXY: https://goproxy.io
      QMAKE_APPLE_DEVICE_ARCHS: arm64
      # SDKROOT: /Library/Developer/CommandLineTools/SDKs/MacOSX11.1.sdk
    steps:

    - name: Install Go
      uses: actions/setup-go@v1
      with:
        go-version: ${{ matrix.go-version }}

    - name: Install Qt for macOS
      uses: jurplel/install-qt-action@v3
      with:
        version: ${{ matrix.qtversion }}
        host: mac
        target: desktop
        dir: ${{ github.workspace }}
        modules: 'qtcharts qtdatavis3d qtpurchasing qtvirtualkeyboard qtwebengine qtnetworkauth qtwebglplugin qtscript'
        setup-python: 'false'
        # mirror: 'http://mirrors.ocf.berkeley.edu/qt/'

```

</details>

See [Downloading logs § Using workflow run logs - GitHub Docs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/using-workflow-run-logs#downloading-logs) to learn how to download the full log.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -43,6 +43,8 @@ jobs:
             requested: "5.15"
           - version: "6.3.2"  # Qt 6.3 is not an LTS version, so '6.3.*' always resolves to '6.3.2'
             requested: "6.3.*"
+          - version: null   # Tools-only build
+            requested: null
         cache:
           - cached
           - uncached
@@ -70,7 +72,7 @@ jobs:
           npm run build
 
       - name: Install Qt5 with options
-        if: startsWith(matrix.qt.version, '5')
+        if: ${{ matrix.qt.version && startsWith(matrix.qt.version, '5') }}
         uses: ./
         with:
           modules: qtwebengine
@@ -79,7 +81,7 @@ jobs:
           cache: ${{ matrix.cache == 'cached' }}
 
       - name: Install Qt6 with options
-        if: startsWith(matrix.qt.version, '6')
+        if: ${{ matrix.qt.version && startsWith(matrix.qt.version, '6') }}
         uses: ./
         with:
           # In Qt 6.2.0+, qtwebengine requires qtpositioning and qtwebchannel
@@ -89,7 +91,7 @@ jobs:
           cache: ${{ matrix.cache == 'cached' }}
 
       - name: Configure test project on windows
-        if: startsWith(matrix.os, 'windows')
+        if: ${{ matrix.qt.version && startsWith(matrix.os, 'windows') }}
         env:
           QT_VERSION: ${{ matrix.qt.version }}
         run: |
@@ -100,7 +102,7 @@ jobs:
         shell: cmd
 
       - name: Configure test project on unix
-        if: (!startsWith(matrix.os, 'windows'))
+        if: ${{ matrix.qt.version && !startsWith(matrix.os, 'windows') }}
         env:
           QT_VERSION: ${{ matrix.qt.version }}
         run: |
@@ -112,3 +114,27 @@ jobs:
           fi
           qmake
         shell: bash
+
+      - name: Install tools with options
+        if: ${{ !matrix.qt.version }}
+        uses: ./
+        with:
+          tools-only: true
+          tools: tools_ifw tools_qtcreator,qt.tools.qtcreator
+          cache: ${{ matrix.cache == 'cached' }}
+
+      - name: Test installed tools
+        if: ${{ !matrix.qt.version }}
+        env:
+          # Conditionally set qtcreator path based on os:
+          QTCREATOR_BIN_PATH: ${{ startsWith(matrix.os, 'macos') && '../Qt/Qt Creator.app/Contents/MacOS/' || '../Qt/Tools/QtCreator/bin/' }}
+        shell: bash
+        run: |
+          # Check if QtIFW is installed
+          ls ../Qt/Tools/QtInstallerFramework/*/bin/
+          ../Qt/Tools/QtInstallerFramework/*/bin/archivegen --version
+  
+          # Check if QtCreator is installed: QtCreator includes the CLI program 'qbs' on all 3 platforms
+          ls "${QTCREATOR_BIN_PATH}"
+          [[ -f "${QTCREATOR_BIN_PATH}qbs" ]] && echo "qbs is installed"
+          "${QTCREATOR_BIN_PATH}qbs" --version || echo "qbs installed but fails to run"

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -31,7 +31,6 @@ jobs:
         os:
           - ubuntu-22.04
           - ubuntu-20.04
-          - ubuntu-18.04
           - windows-2022
           - windows-2019
           - macos-11
@@ -49,10 +48,6 @@ jobs:
           - cached
           - uncached
 
-        # Ubuntu 18 is not a supported target for Qt 6: https://www.qt.io/blog/qt6-development-hosts-and-targets
-        exclude:
-          - os: ubuntu-18.04
-            qt: {version: "6.3.2", requested: "6.3.*"}
     steps:
       - uses: actions/checkout@v3
 
@@ -136,5 +131,4 @@ jobs:
   
           # Check if QtCreator is installed: QtCreator includes the CLI program 'qbs' on all 3 platforms
           ls "${QTCREATOR_BIN_PATH}"
-          [[ -f "${QTCREATOR_BIN_PATH}qbs" ]] && echo "qbs is installed"
-          "${QTCREATOR_BIN_PATH}qbs" --version || echo "qbs installed but fails to run"
+          "${QTCREATOR_BIN_PATH}qbs" --version

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -36,10 +36,13 @@ jobs:
           - windows-2019
           - macos-11
           - macos-12
-        version:
-          - "5.9.0"
-          - "5.15.2"
-          - "6.2.0"
+        qt:
+          - version: "5.9.0"
+            requested: "5.9.0"
+          - version: "5.15.2"
+            requested: "5.15"
+          - version: "6.3.2"  # Qt 6.3 is not an LTS version, so '6.3.*' always resolves to '6.3.2'
+            requested: "6.3.*"
         cache:
           - cached
           - uncached
@@ -47,7 +50,7 @@ jobs:
         # Ubuntu 18 is not a supported target for Qt 6: https://www.qt.io/blog/qt6-development-hosts-and-targets
         exclude:
           - os: ubuntu-18.04
-            version: "6.2.0"
+            qt: {version: "6.3.2", requested: "6.3.*"}
     steps:
       - uses: actions/checkout@v3
 
@@ -67,28 +70,28 @@ jobs:
           npm run build
 
       - name: Install Qt5 with options
-        if: startsWith(matrix.version, '5')
+        if: startsWith(matrix.qt.version, '5')
         uses: ./
         with:
           modules: qtwebengine
-          version: ${{ matrix.version }}
+          version: ${{ matrix.qt.requested }}
           tools: tools_ifw tools_qtcreator,qt.tools.qtcreator
           cache: ${{ matrix.cache == 'cached' }}
 
       - name: Install Qt6 with options
-        if: startsWith(matrix.version, '6.2')
+        if: startsWith(matrix.qt.version, '6.3')
         uses: ./
         with:
           # In Qt 6.2.0, qtwebengine requires qtpositioning and qtwebchannel
           modules: qtwebengine qtpositioning qtwebchannel
-          version: ${{ matrix.version }}
+          version: ${{ matrix.qt.requested }}
           tools: tools_ifw tools_qtcreator,qt.tools.qtcreator
           cache: ${{ matrix.cache == 'cached' }}
 
       - name: Configure test project on windows
         if: startsWith(matrix.os, 'windows')
         env:
-          QT_VERSION: ${{ matrix.version }}
+          QT_VERSION: ${{ matrix.qt.version }}
         run: |
           cd tests/TestWithModules
           for /f "delims=" %%d in ( 'vswhere.exe -latest -property installationPath' ) do @( call "%%d\VC\Auxiliary\Build\vcvars64.bat" )
@@ -99,7 +102,7 @@ jobs:
       - name: Configure test project on unix
         if: (!startsWith(matrix.os, 'windows'))
         env:
-          QT_VERSION: ${{ matrix.version }}
+          QT_VERSION: ${{ matrix.qt.version }}
         run: |
           cd tests/TestWithModules
           if [[ $QT_VERSION == 6* ]]; then

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -79,10 +79,10 @@ jobs:
           cache: ${{ matrix.cache == 'cached' }}
 
       - name: Install Qt6 with options
-        if: startsWith(matrix.qt.version, '6.3')
+        if: startsWith(matrix.qt.version, '6')
         uses: ./
         with:
-          # In Qt 6.2.0, qtwebengine requires qtpositioning and qtwebchannel
+          # In Qt 6.2.0+, qtwebengine requires qtpositioning and qtwebchannel
           modules: qtwebengine qtpositioning qtwebchannel
           version: ${{ matrix.qt.requested }}
           tools: tools_ifw tools_qtcreator,qt.tools.qtcreator

--- a/action/src/main.ts
+++ b/action/src/main.ts
@@ -30,6 +30,30 @@ const execPython = async (command: string, args: readonly string[]): Promise<num
   return exec(`${python} -m ${command} ${args.join(" ")}`);
 };
 
+const locateQtArchDir = (installDir: string): string => {
+  // For 6.4.2/gcc, qmake is at 'installDir/6.4.2/gcc_64/bin/qmake'.
+  // This makes a list of all the viable arch directories that contain a qmake file.
+  const qtArchDirs = glob
+    .sync(`${installDir}/[0-9]*/*/bin/qmake*`)
+    .map((s) => s.replace(/\/bin\/qmake[^/]*$/, ""));
+
+  // For Qt6 mobile and wasm installations, a standard desktop Qt installation
+  // must exist alongside the requested architecture.
+  // In these cases, we must select the first item that ends with 'android*', 'ios', or 'wasm*'.
+  const requiresParallelDesktop = qtArchDirs.filter((p) =>
+    p.match(/6\.\d+\.\d+\/(android[^/]*|ios|wasm[^/]*)$/)
+  );
+  if (requiresParallelDesktop.length) {
+    // NOTE: if multiple mobile/wasm installations coexist, this may not select the desired directory
+    return requiresParallelDesktop[0];
+  } else if (!qtArchDirs.length) {
+    throw Error(`Failed to locate a Qt installation directory in  ${installDir}`);
+  } else {
+    // NOTE: if multiple Qt installations exist, this may not select the desired directory
+    return qtArchDirs[0];
+  }
+};
+
 class Inputs {
   readonly host: "windows" | "mac" | "linux";
   readonly target: "desktop" | "android" | "ios";
@@ -170,12 +194,6 @@ class Inputs {
     this.aqtVersion = core.getInput("aqtversion");
 
     this.py7zrVersion = core.getInput("py7zrversion");
-  }
-
-  public get versionDir(): string {
-    // Weird naming scheme exception for qt 5.9
-    const version = this.version === "5.9.0" ? "5.9" : this.version;
-    return nativePath(`${this.dir}/${version}`);
   }
 
   public get cacheKey(): string {
@@ -331,12 +349,12 @@ const run = async (): Promise<void> => {
     }
 
     // Set environment variables
-    const qtPath = nativePath(glob.sync(`${inputs.versionDir}/**/*`)[0]);
     if (inputs.setEnv) {
       if (inputs.tools.length) {
         core.exportVariable("IQTA_TOOLS", nativePath(`${inputs.dir}/Tools`));
       }
       if (!inputs.toolsOnly) {
+        const qtPath = nativePath(locateQtArchDir(inputs.dir));
         if (process.platform === "linux") {
           setOrAppendEnvVar("LD_LIBRARY_PATH", nativePath(`${qtPath}/lib`));
         }

